.eslintignore

```diff
@@ -2,3 +2,4 @@ jest.config.ts
 license-header.js
 rollup.config.js
 tsconfig.json
+e2e/browser/test-app
```

Comment:
Why add the test app to the unlinted files? Does it prevent conflicts with its own linter config?